apis_bibsonomy/views.py

```diff
@@ -0,0 +1,42 @@
+from django.views.generic.list import ListView
+from django.views.generic.detail import DetailView
+from django.views.generic.edit import DeleteView
+from django.urls import reverse_lazy
+
+from .models import Reference
+
+
+class ReferenceDetailView(DetailView):
+    model = Reference
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        # we collect a list of other references instances, that point to the
+        # same reference, and add that to the context
+        obj = self.get_object()
+        similar_references = Reference.objects.filter(
```

Comment:
I think this is something that needs some way of configuration for other projects. For some projects it might be interesting to have all references as similar that point to the same zotero/bibsonomy item (because they are using a large set of sources), for others filtering also for page ranges and/or folio might make more sense.

Reply:
I agree - I have now added a setting `REFERENCE_SIMILARITY` that receives a list of fields that should be equal. I think even more customization (page *ranges*) would mean writing a custom filter - I've moved the filtering to a `get_similar_objects` method of the view, so projects can inherit from the view and override the method.